.github/workflows/php8-compatibility.yml

```
@@ -43,10 +38,10 @@ jobs:
       with:
         php-version: '7.4'
         coverage: none
-        tools: prestissimo, composer:v2
+        tools: composer:v2
 
     - name: Install dependencies
       run: composer install
 
     - name: Check PHP Compatibility
-      run: ./vendor/bin/phpcs -p ads-txt.php inc --standard=PHPCompatibilityWP --extensions=php --runtime-set testVersion ${{ matrix.php }}
+      run: ./vendor/bin/phpcs --standard=.phpcompat.xml.dist
```

```suggestion
      run: ./vendor/bin/phpcs -p ads-txt.php inc --standard=PHPCompatibilityWP --extensions=php --runtime-set testVersion 7.4-
```

Is there a reason you've created a new file rather than specify in the version range here?

---

@peterwilsoncc yes. I think having config files separate instead of keeping long commands inline is cleaner, more readable and also makes it easier to update them, if needed. What do you think?

---

Yep, that's cool. I just wanted to make sure it was by design.